lib/runtime/wasmer/imports.go

```diff
@@ -1920,8 +1920,8 @@ func ext_storage_exists_version_1(context unsafe.Pointer, keySpan C.int64_t) C.i
 	key := asMemorySlice(instanceContext, keySpan)
 	logger.Debugf("key: 0x%x", key)
 
-	val := storage.Get(key)
-	if len(val) > 0 {
+	value := storage.Get(key)
```

Comment:
Wondering if we could return an error from `Get(key)` method like we do for database `Get` method where it returns `ErrKeyNotFound`

Reply:
Substrate returns a result with byte array or `None`, I think the analogy `[]byte(nil)` <-> `None` fits fine, especially since it also matches the implementation (`StorageValue = nil` in the trie code).

Reply:
Yeah, I thought would be more golang-ish this way:

```go
value, err := storage.Get(key)
if errors.Is(err, ErrKeyNotFound) {}
```
but that was just a thought and I belive it will require a more effort/deltas to accomplish so I am good with the current changes

Reply:
Yeah I'm kinda shared too :smile: 
But since we now (finally duh) have a clear understanding `nil` means empty in the trie implementation/interface/spec (unlike `[]byte{}`), I guess we can keep it as is too.